shipment_advice_planner_toursolver/models/toursolver_backend.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class TourSolverBackend(models.Model):
+    _name = "toursolver.backend"
+    _description = "TourSolver Backend"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    resource_properties_definition = fields.PropertiesDefinition(
+        string="Resource Properties"
+    )
+    url = fields.Char()
+    api_key = fields.Char()
+    delivery_window_disabled = fields.Boolean()
+    delivery_window_start = fields.Float(default=8.0)
+    delivery_window_end = fields.Float(default=17.0)
+    delivery_duration = fields.Integer(
+        string="Fixed time spent delivering a customer",
+        help="Duration in seconds needed to deliver a customer",
+        default=180,
+    )
+    duration = fields.Integer(
+        string="Optimization process max duration",
+        help="Duration in seconds allowed to the computation of the optimization",
+    )
+    loading_duration = fields.Integer(
+        string="Fixed initial loading time", help="Loading time in minutes"
+    )
+    work_penalty = fields.Float(
+        "Fixed cost working/hour", help="The cost of a resource working for an hour."
+    )
+    travel_penalty = fields.Float(
+        "Fixed cost travelling/hour",
+        help="The cost for a resource of driving for one distance unit.",
```

Comment:
These fields are default values for resource definition in toursolver. That means that if the property is specified on the resource itself, these values here should not orverride the one on the resources